internal/sdkprovider/service/clickhouse/clickhouse_database.go

```diff
@@ -77,6 +76,12 @@ func resourceClickhouseDatabaseRead(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(schemautil.ResourceReadHandleNotFound(err, d))
 	}
 
+	if err := d.Set("project", projectName); err != nil {
```

Comment:
Should we add `termination_protection` field as well?

Reply:
It is a virtual field. It doesn't exist in the API. Really confusing. I have also removed the default value, which false, as it does change the state, while it makes no sense to have false default value for a bool field.

Reply:
Yes was confusing me when i import and it's import with a false value even i set true in the destination ressource. but it's not a blocker as the import block do the change, but it's a confusing drift

Reply:
@jean-mercier-hivebrite If you set the `termination_protection` in the destination explicitly then the state should be valid. Please let us know if you have any issues with it.

Reply:
i'm use to use the import block it's make the import and the plan at once 

`import {
  id = "${var.aiven_project}/${var.aiven_clickhouse_service_name}/db_name"
  to = aiven_clickhouse_database.db_name
}`

And in this case with the previous behaviour you have import and change to put terminaison protection to "true"

you have the same problem on kafka_topic import by the way

Reply:
but it's not a blocker